testmod-fabric/build.gradle

```diff
@@ -21,10 +21,16 @@ configurations {
     developmentFabric.extendsFrom common
 }
 
+repositories {
+    maven { url "https://maven.terraformersmc.com/releases/" }
+}
+
 dependencies {
     modImplementation "net.fabricmc:fabric-loader:${rootProject.fabric_loader_version}"
     modImplementation "net.fabricmc.fabric-api:fabric-api:${rootProject.fabric_api_version}"
 
+    modLocalRuntime("com.terraformersmc:modmenu:${rootProject.mod_menu_version}") { transitive false }
```

Comment:
I would not add modmenu as a runtime dep because they usually port later than us and their api is pretty much never broken anyways, so I just expect it to work